api/src/main/java/org/openmrs/module/sync/SyncUtil.java

```diff
@@ -1371,11 +1371,8 @@ public static String formatTransactionStatus(Transaction tx) {
 		if (tx.isActive()) {
 			return "TX ACTIVE";
 		}
-		if (tx.wasCommitted()) {
-			return "TX COMMITTED";
-		}
-		if (tx.wasRolledBack()) {
-			return "TX ROLLED BACK";

```

Comment:
The above methods are not available in Hibernate from 2.5.x, and other statuses are added.